iis_action.ps1

```diff
@@ -0,0 +1,53 @@
+Param(
+    [parameter(Mandatory = $true)]
+    [string]$server,
+    [parameter(Mandatory = $true)]
+    [string]$website_name,
+    [parameter(Mandatory = $true)]
+    [string]$user_id,
+    [parameter(Mandatory = $true)]
+    [SecureString]$password,
+    [parameter(Mandatory = $true)]
+    [string]$cert_path
+)
+
+$display_action = 'IIS Website Status'
+$display_action_past_tense = "$display_action$past_tense Returned"
+
+Write-Output $display_action
+
+$credential = [PSCredential]::new($user_id, $password)
+$so = New-PSSessionOption -SkipCACheck -SkipCNCheck -SkipRevocationCheck
+
+Write-Output "Importing remote server cert..."
+Import-Certificate -Filepath $cert_path -CertStoreLocation 'Cert:\LocalMachine\Root'
+
+$script = {
+    # Relies on WebAdministration Module being installed on the remote server
+    # This should be pre-installed on Windows 2012 R2 and later
+    # https://docs.microsoft.com/en-us/powershell/module/?term=webadministration
+
+    $website = Get-IISSite -Name $Using:website_name
+    if (!$website) {
+        Return 1
+        Exit
+    }
+
+    Return "$($website.Name): $($website.State)"
```

Comment:
Same comment as the other repo, since you're concatenating items here you should make note of this on the readme and the action.yml.  I think since people are going to be using this value programmatically it might be better to just return the state, especially since they already know the site it was for.